collectors/bzimport/tests/test_convertors.py

```diff
@@ -1612,6 +1619,32 @@ def test_summary_meta_attr(self):
         assert "bz_summary" in flaw.meta_attr
         assert flaw.meta_attr["bz_summary"] == flaw_bug["summary"]
 
+    def test_fixed_in_meta_attr(self):
+        """
+        Test that fixed_in bz field is saved as-is in Flaw.meta_attr
+
+        The model structure of CVEv5Package and CVEv5Ver doesn't store the
```

Comment:
There should be probably `Package` and `PackageVer`.

Reply:
Fixed in commit [266dca9](https://github.com/RedHatProductSecurity/osidb/pull/307/commits/266dca9466f9c8a6c8283a42e86e532a5a98b558)